app/cinema/hall.py

```diff
@@ -0,0 +1,18 @@
+from app.cinema.customer import Customer
+from app.cinema.cinema_stuff import Cleaner
+
+
+class CinemaHall:
+
+    def __init__(self, number: int) -> None:
+        self.number = number
+
+    def movie_session(
+            self, movie_name: str, customer: list, cleaning_stuff: Cleaner
```

Comment:
Specify the list type
```suggestion
            self, movie_name: str, customer: list[Customer], cleaning_stuff: Cleaner
```